BasicRdl/ViewModels/Dialogs/EnumerationValueDefinitionDialogViewModel.cs

```diff
@@ -56,38 +68,60 @@ public EnumerationValueDefinitionDialogViewModel()
         /// The text Parameter Type.
         /// </param>
         /// <param name="transaction">
-        /// The <see cref="ThingTransaction"/> that contains the log of recorded changes.
+        /// The <see cref="ThingTransaction" /> that contains the log of recorded changes.
         /// </param>
         /// <param name="session">
-        /// The <see cref="ISession"/> in which the current <see cref="Thing"/> is to be added or updated
+        /// The <see cref="ISession" /> in which the current <see cref="Thing" /> is to be added or updated
         /// </param>
         /// <param name="isRoot">
-        /// Assert if this <see cref="EnumerationValueDefinitionDialogViewModel"/> is the root of all <see cref="IThingDialogViewModel"/>
+        /// Assert if this <see cref="EnumerationValueDefinitionDialogViewModel" /> is the root of all
+        /// <see cref="IThingDialogViewModel" />
         /// </param>
         /// <param name="dialogKind">
-        /// The kind of operation this <see cref="EnumerationValueDefinitionDialogViewModel"/> performs
+        /// The kind of operation this <see cref="EnumerationValueDefinitionDialogViewModel" /> performs
         /// </param>
         /// <param name="thingDialogNavigationService">
-        /// The <see cref="IThingDialogNavigationService"/> that is used to navigate to a dialog of a specific <see cref="Thing"/>.
+        /// The <see cref="IThingDialogNavigationService" /> that is used to navigate to a dialog of a specific
+        /// <see cref="Thing" />.
         /// </param>
         /// <param name="container">
-        /// The <see cref="Thing"/> that contains the created <see cref="Thing"/> in this Dialog
+        /// The <see cref="Thing" /> that contains the created <see cref="Thing" /> in this Dialog
         /// </param>
         /// <param name="chainOfContainers">
-        /// The optional chain of containers that contains the <paramref name="container"/> argument
+        /// The optional chain of containers that contains the <paramref name="container" /> argument
         /// </param>
         /// <exception cref="ArgumentException">
-        /// The container must be of type <see cref="ReferenceDataLibrary"/>.
+        /// The container must be of type <see cref="ReferenceDataLibrary" />.
         /// </exception>
         public EnumerationValueDefinitionDialogViewModel(EnumerationValueDefinition enumerationValueDefinition, IThingTransaction transaction, ISession session, bool isRoot, ThingDialogKind dialogKind, IThingDialogNavigationService thingDialogNavigationService, Thing container = null, IEnumerable<Thing> chainOfContainers = null)
             : base(enumerationValueDefinition, transaction, session, isRoot, dialogKind, thingDialogNavigationService, container, chainOfContainers)
         {
         }
+
+        /// <summary>
+        /// Gets or sets the ShortName
+        /// </summary>
+        [ValidationOverride(true, "RDLShortName")]
```

Comment:
Might be better to add a new one in ValidationService.ValidationMap specifically for this one instead of using an existing key. New key would be "EnumerationValueDefinitionShortName".